tools/differ/differ.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl import app
+from absl import flags
+from absl import logging
+import pandas as pd
+import helper
+import os
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('current_data', '', 'Path to the current MCF data.')
+flags.DEFINE_string('previous_data', '', 'Path to the previous MCF data.')
+flags.DEFINE_string('output_location', 'results', 'Path to the output data.')
```

Comment:
If this is expected to be a folder, pls add to the docstring
'Path to the output data folder.'

Reply:
Done